### PR TITLE
added zipcode to user registration form

### PR DESCRIPTION
**Pull request by ehamp**
@aharelick 

Added the zip code to the user registration form, it uses the helper methods to get the information from the database

Waiting for the other profile stuff to be merged, so that I can do the profile side of the location task

### CODE REVIEW

File: app/account/forms.py [hunk 1]
```diff
@@ -42,6 +42,10 @@ class RegistrationForm(Form):
         EqualTo('password2', 'Passwords must match')
     ])
     password2 = PasswordField('Confirm password', validators=[InputRequired()])
+    zip_code = StringField('ZIP Code', validators=[
+        InputRequired(),
+        Length(1, 6)
```
Comment (aharelick) [inline app/account/forms.py]: we don't want a range from 1 to 6. This should be changed to `Length(5, 5)`.

File: app/account/views.py [hunk 2]
```diff
@@ -40,10 +40,12 @@ def register():
     """Register a new user, and send them a confirmation email."""
     form = RegistrationForm()
     if form.validate_on_submit():
+        zip_code = ZIPCode.create_zip_code(form.zip_code.data)
         user = User(first_name=form.first_name.data,
                     last_name=form.last_name.data,
                     email=form.email.data,
-                    password=form.password.data)
+                    password=form.password.data,
+                    zip_code_id=zip_code.id)
```
Comment (aharelick) [inline app/account/views.py:48]: this totally works but I think we should start using `zip_code=zip_code` instead (should do the same thing), let me know if you disagree.

Comment (aharelick) [inline app/account/views.py:48]: nvm, this is fine, let's keep this the way it is. I can explain the thoughts that happened if you'd like in person.

**Comment by aharelick** [comment]
just one comment about the `Length` validator, otherwise looks great to me. Let me know when that's fixed

**Comment by ehamp** [comment]
@aharelick  fixed the length!

**Comment by aharelick** [comment]
lgtm, let's merge it

**Pull request merged**